system/Config/BaseService.php

```diff
@@ -173,9 +173,9 @@ class BaseService
      *
      * $key must be a name matching a service.
      *
-     * @param mixed ...$params
+     * @param array|object|string|null ...$params
```

Comment:
These are parameters for services. Maybe all except for resource?
Or resource might be passed by devs?

Reply:
I have been added!